universal_transfer_operator/src/universal_transfer_operator/data_providers/database/base.py

```diff
@@ -517,6 +541,41 @@ def load_file_to_table(
             if_exists="append",
             chunk_size=chunk_size,
         )
+        return self.get_table_qualified_name(output_table)
+
+    def load_dataframe_to_table(
+        self,
+        input_dataframe: pd.DataFrame,
+        output_table: Table,
+        if_exists: LoadExistStrategy = "replace",
+        chunk_size: int = DEFAULT_CHUNK_SIZE,
+        columns_names_capitalization: ColumnCapitalization = "original",
+    ) -> str:
+        """
+        Load content of dataframe in output_table.
+
+        :param input_dataframe: dataframe
+        :param output_table: Table to create
+        :param if_exists: Overwrite file if exists
+        :param chunk_size: Specify the number of records in each batch to be written at a time
+        :param normalize_config: pandas json_normalize params config
+        :param columns_names_capitalization: determines whether to convert all columns to lowercase/uppercase
+            in the resulting dataframe
+        """
+
+        self.create_schema_and_table_if_needed_from_dataframe(
```

Comment:
```suggestion
        self.create_schema_and_table_if_needed(
```

thats a big method name IMO, WDYT abt the suggestion?

Reply:
@phanikumv We have two ways of creating the schema via file and dataframe, that's why this name and there is already a method which has this name which uses files. But I'm open to other suggestions.